frontend/src/__tests__/cypress/cypress/pages/servingRuntimes.ts

```diff
@@ -5,27 +5,27 @@ class ServingRuntimeRow {
   constructor(public readonly id: string) {}
 
   find() {
-    return cy.get(`#${this.id}`);
+    return cy.findByTestId(`${this.id}`);
   }
 
   shouldBeMultiModel(enabled = true) {
     this.find()
       .findByRole('list', { name: 'Label group category' })
```

Comment:
Is it possible to add a data-testid here otherthan finding by role

Reply:
This is not the static selector, so I am not able to add data-testid for them